src/locales/de/app.json

```diff
@@ -0,0 +1,132 @@
+{
+	"loading": "Lade...",
+	"download-ready": "Jetzt herunterladen!",
+	"navigation": {
+		"home": "Start",
+		"configuration": "Kalender erzeugen",
+		"features": "Features",
+		"faq": "FAQ"
+	},
+	"language": {
+		"label": "Sprache",
+		"english": "Englisch",
+		"polish": "Polnisch",
+		"french": "Französisch",
+		"spanish": "Spanisch",
+		"german": "Deutsch"
+	},
+	"configuration": {
+		"selector": {
+			"label": "Start",
+			"template": {
+				"label": "Wähle eine Vorlage aus:",
+				"basic": "Einfach",
+				"advanced": "Fortschritten",
+				"blank": "Leer",
+				"minimalistic": "Minimalistisch"
+			},
+			"upload": {
+				"label": "Lade einen existierenden ReCalendar PDF hoch um die Konfiguration zu übernehmen:",
+				"loading": "Lese Konfiguration aus der PDF Datei, bitte warten...",
```

Comment:
"PDF Datei" is redundant, I suggest: "Lese Konfiguration aus dem PDF, bitte warten..."

Reply:
OK .. both is OK with me.